packages/security/src/DefaultCredentialManager.ts

```diff
@@ -114,12 +97,27 @@ export class DefaultCredentialManager extends AbstractCredentialManager {
     // the abstract class initialization in the future.
     super(service, displayName);
 
-    // Gather all services
-    this.allServices = JSON.parse(JSON.stringify(this.ALTERNATIVE_SERVICES));
-    if (this.allServices.indexOf(this.service) === -1) {
-      this.allServices.push(this.service);
+    /* Gather all services. We will load secure properties for the first
+     * successful service found in the order that they are placed in this array.
+     */
+    this.allServices = [this.service];
+
+    /* In our current implementation, this.service is always
+     * DefaultCredentialManager.SVC_NAME. We keep the logic below,
+     * in case we re-enable overrides in the future.
+     */
+    if (this.service !== DefaultCredentialManager.SVC_NAME) {
+        this.allServices.push(DefaultCredentialManager.SVC_NAME);
     }
-    this.allServices.push(DefaultCredentialManager.SVC_NAME);
+
+    /* Previous services under which we will look for credentials.
+     * We dropped @brightside/core because we no longer support the
+     * lts-incremental version of the product.
+     */
```

Comment:
Agree that it makes sense to drop @brightside/core 👍 

Should we also drop the logic below (lines 186-193) that migrated old @brightside/core property names (username -> user, pass -> password)? Perhaps the `loadHelper` defined on line 184 is unnecessary and `this.getCredentialHelper` can be directly called instead.

Reply:
If the only case for conversion from username -> user, and pass -> password are in LTS-incremental, then I agree that it makes sense to remove them.

I will wait to let other reviewer confirm this, then I will add an update to this PR to remove that logic.